assets/queries/openAPI/2.0/unused_schema_definition/test/negative1.json

```diff
@@ -0,0 +1,48 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Simple API Overview",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/": {
+      "get": {
+        "operationId": "listVersionsv2",
+        "summary": "List API versions",
+        "responses": {
+          "200": {
+            "description": "200 response"
+          }
+        },
+        "parameters": [
+          {
+            "name": "category",
+            "in": "body",
+            "description": "max records to return",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Category"
+            },
+            "desc": {
```

Comment:
`desc` is not a valid value. Maybe we could delete it. The same happens in the other samples